### DISCUSSION TITLE
Support for load balancing using environment variable PGLOADBALANCEHOSTS=random

### DISCUSSION THREAD

**Discussion by mkleczek** [PostgREST/postgrest#3593] [Q&A] [answered]
Hi, thanks for the latest release!

Got a question:
Since https://github.com/PostgREST/postgrest/issues/3414 is fixed - does the latest version support load balancing using multiple host names in PGHOST and PGLOADBALANCEHOSTS=random?

**Comment by wolfgangwalther** [comment]
IIRC, no - we just made sure to solve that issue in a way that will allow us to add this load balancing in the future, but didn't add it, yet.

**Reply by mkleczek** [reply]
This should be handled automatically by libpq though as long as its version is >=16.

What version of libpq is PostgREST using at the moment?

**Reply by wolfgangwalther** [reply, marked as answer]
The static executable / docker image uses libpq v16: nix/overlays/postgresql-libpq.nix

The dynamically linked release artifacts ofc use whatever is installed.

You might be right that load balancing actually works - but it would have worked before already (assuming you had libpq 16). I don't think anyone tried, though.
